src/lib/y2storage/lvm_pv.rb

```diff
@@ -55,6 +58,27 @@ def plain_blk_device
       blk_device.plain_device
     end
 
+    # Whether the PV is orphan (not associated to any VG)
+    #
+    # @return [Boolean]
+    def orphan?
+      lvm_vg.nil?
+    end
+
+    # Display name to represent the PV
+    #
+    # Only orphan PV has its own representation.
+    #
+    # @return [String, nil]
+    def display_name
+      return nil unless orphan?
+
+      textdomain "storage"
+
+      # TRANSLATORS: display name when the PV has no associated VG
+      format(_("Orphan PV on %{device}"), device: plain_blk_device.name)
```

Comment:
Not sure if "Orphan PV" would mean something to most people out there. What about "Unused LVM PV on %s"?

Reply:
New column type is using "Orphan PV". Should we "Unused LVM PV" instead?

Reply:
I think so